lib/node_modules/@stdlib/stats/base/snanmeanwd/test/test.ndarray.js

```diff
@@ -106,15 +104,13 @@ tape( 'the function supports a `stride` parameter', function test( t ) {
 		NaN   // 4
 	]);
 
-	N = floor( x.length / 2 );
-	v = snanmeanwd( N, x, 2, 0 );
+	v = snanmeanwd( 4, x, 2, 0 );
```

Comment:
```suggestion
	v = snanmeanwd( 5, x, 2, 0 );
```
you should include the `NaN` element in the computation so that we can verify if our implementation is handling it correctly or not